elasticmock/fake_elasticsearch.py

```diff
@@ -216,7 +216,7 @@ def suggest(self, body, index=None, params=None):
             raise NotFoundError(404, 'IndexMissingException[[{0}] missing]'.format(index))
 
         result_dict = {}
-        for key, value in body.iteritems():
+        for key, value in body.items():
```

Comment:
This change will break Python 2.7 code. Could you verify if you're running into Python 3 or Python 3 before?